src/features/import-maps.js

```diff
@@ -50,10 +53,12 @@ function processScripts () {
 }
 
 function extendImportMap (importMap, newMapText, newMapUrl) {
+  var newMap = {};
   try {
-    var newMap = JSON.parse(newMapText);
+    newMap = JSON.parse(newMapText);
   } catch (err) {
-    throw Error(process.env.SYSTEM_PRODUCTION ? errMsg(1) : errMsg(1, "systemjs-importmap contains invalid JSON"));
+    console.warn(Error(process.env.SYSTEM_PRODUCTION ? errMsg('W5') : errMsg('W5', "systemjs-importmap contains invalid JSON")));
+    console.warn({invalidJsonString: newMapText});
```

Comment:
On second thoughts here, it's probably preferable just to combine the map into the message, like any type of error frame.

Ideally we would just include a few lines around the exact issue, but since we know that's what we want to do eventually, it doesn't seem _so_ bad to me to just include everything.

```suggestion
    console.warn(Error(process.env.SYSTEM_PRODUCTION ? errMsg('W5') : errMsg('W5', "systemjs-importmap contains invalid JSON:\n\n" + newMapText)));
```

Reply:
Done.